SessionMessagingKit/Shared Models/SessionThreadViewModel.swift

```diff
@@ -2067,6 +2073,7 @@ public extension SessionThreadViewModel {
                 
                 \(contact[.rowId]) AS \(ViewModel.Columns.rowId),
                 \(contact[.id]) AS \(ViewModel.Columns.threadId),
+                \(contact[.isApproved]) AS \(ViewModel.Columns.isContactApproved),
```

Comment:
This results in an invalid query due to how the `ScopeAdapter` works - you need to update `numColumnsBeforeProfiles` to be `9` since you've added a new column before an `allColumns` value (otherwise the index is off by 1)